### PR TITLE
Run perf tests without separate `package.json`

### PR DESCRIPTION
**Pull request by nrfox**
This change removes the separate `package.json` and in the process adds a couple large dev dependencies in the form of cypress webpack plugins. The extra plugins were necessary to tell cypress where the typescript binary was located. If this can be achieved without the plugins, that would be even better. I'm not confident this is the most optimal way to setup all these configs without needing a separate `package.json` for the perf tests but this was the way that worked. Much of the frontend tooling assumes that you are running underneath the package root at `./kiali/frontend` but since the tests run at `./kiali/tests/perf`, this is not the case. An alternative might be to just move the tests underneath `./kiali/frontend`.

In summary:
- Adds a few makefile commands to make running the performance tests easier
- Removes the separate `package.json` for the perf tests
- Adds a couple large dev dependencies for pre-processing the cypress config

### CODE REVIEW

**Comment by nrfox** [comment]
Moved everything under frontend folder at `frontend/cypress/perf`. IMO this is the simplest setup as we can share deps without having a separate `package.json` but lmk what you think.

**Comment by josunect** [comment]
Tested changes and working correctly.
